connectors-api/src/main/ruby/connectors_exception_tracking.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+
+# frozen_string_literal: true
+
+require 'swiftype_exception_tracking' unless defined?(Rails)
+require 'bson'
+require_relative './connectors_logger'
+
+java_package 'co.elastic.connectors.api'
+
+class ConnectorsExceptionTracking
```

Comment:
Should this whole class be inlined to `Swiftype::ExceptionTracking`?

It seems like this class is only wrapping the `Swiftype::ExceptionTracking` class, while `Swiftype::ExceptionTracking` seem like it's not meant to be used directly.

Reply:
Yep, that's exactly the case. We don't want to move the _real_ Swiftype::ExceptionTracking into this repo though. The only reason it's included in `connectors-stubs` is so that this code can still function in ent-search (without `connectors-stubs`) and also run in the MVP for the external framework (with `connectors-stubs`). In the long run, we should do some refactoring and cleaning here, and replace `connectors-stubs` with more fully-fleged implementations. But right now, as we cut scope, those classes can practically no-op as long as they don't cause runtime errors.